phpstan.neon.dist

```diff
@@ -11,6 +11,9 @@ parameters:
 
     # Make tests analyses working with Symfony's PHPUnit bridge
     bootstrapFiles:
-       - vendor/bin/.phpunit/phpunit-7.5-0/vendor/autoload.php
+       - vendor/bin/.phpunit/phpunit-9.5-0/vendor/autoload.php
 
-    ignoreErrors: []
+    ignoreErrors:
+        -
+         message: '#Service "session" is not registered in the container.#'
+         path: tests/Controller/SessionPrepareTrait.php
```

Comment:
Note: this is temporary, we will need to fix how the session is initialized in tests before bumping to symfony 6